proto/greenfield/storage/events.proto

```diff
@@ -424,3 +426,9 @@ message EventMirrorGroupResult {
     (gogoproto.nullable) = false
   ];
 }
+
+// EventStalePolicyCleanup is emitted when specified block height's stale policies is garbage collected
+message EventStalePolicyCleanup {
+  int64 blockNum = 1;
+  DeleteInfo delete_info =2;
```

Comment:
It's important to note that the meta-service may require these events in order to delete its policy. cc @krish-nr @ruojunm